packages/replace/test/sourcemaps.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable consistent-return, global-require, import/no-dynamic-require */
```

Comment:
It doesn't look like `global-require` and `import/no-dynamic-require` are disabling any warnings or errors in this file.

Reply:
You're right - I fixed this in misc.js as well